src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -51,32 +53,50 @@ const NavHeader = (props) => {
     }
   };
 
-  const onDomainLayout = (event) => {
-    const {height} = event.nativeEvent.layout;
-    setDomainHeight(height);
-  };
+  const onDomainLayout = React.useCallback(
+    (event) => {
+      const {height} = event.nativeEvent.layout;
+      setDomainHeight(Math.round(height));
+    },
+    [setDomainHeight]
+  );
 
   const backScreen = () => {
     navigation.goBack();
   };
 
   const getBottomPostition = () => {
-    let bottom = 0;
+    let containerHeight = 0;
     if (isHeaderHide) {
-      bottom = dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT2;
+      containerHeight = dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT2;
     } else {
-      bottom = hideSeeMember
-        ? dimen.size.TOPIC_FEED_HEADER_HEIGHT
-        : dimen.size.TOPIC_FEED_HEADER_HEIGHT + normalize(4);
+      containerHeight =
+        hideSeeMember || !isFollow
+          ? dimen.size.TOPIC_FEED_HEADER_HEIGHT
+          : dimen.size.TOPIC_FEED_HEADER_HEIGHT + normalize(4);
     }
-    return (bottom - domainHeight) / 2;
+    return Math.round((containerHeight - domainHeight) / 2);
+  };
+
+  const heightWithCoverImage = () => {
+    if (coverPath) {
+      return {height: dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT_COVER};
+    }
+    return null;
   };
 
   return (
     <Animated.View style={{height: animatedHeight}}>
-      <View style={[styles.Nav(isHeaderHide)]}>
+      <ImageBackground
+        source={{uri: coverPath}}
+        style={[styles.Nav(isHeaderHide), heightWithCoverImage()]}
+        imageStyle={{opacity: isHeaderHide ? 0 : 1}}>
         <TouchableOpacity onPress={() => backScreen()} style={styles.backbutton}>
-          <MemoIcArrowBack width={normalize(24)} height={normalize(24)} />
+          {coverPath && !isHeaderHide ? (
+            <MemoIcArrowBackCircle width={normalize(32)} height={normalize(32)} />
+          ) : (
+            <MemoIcArrowBack width={normalize(24)} height={normalize(24)} />
+          )}
         </TouchableOpacity>
         <View style={styles.containerAction}>
           {!isFollow && isHeaderHide ? (
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

> Note: This review was outside of the patch, so it was mapped to the patch with the greatest overlap. Original lines [89-110]

The outer `<View>` has been replaced with `<ImageBackground>`, which is used to display the cover image. The back button icon is replaced with `MemoIcArrowBackCircle` when a cover image is present and the header is not hidden. The action container's background color is replaced with transparent. These changes are consistent with the goal of adding a cover image feature.
```diff
- <Animated.View style={{height: animatedHeight}}>
-   <View style={[styles.Nav(isHeaderHide)]}>
+ <Animated.View style={{height: animatedHeight}}>
+   <ImageBackground
+     source={{uri: coverPath}}
+     style={[styles.Nav(isHeaderHide), heightWithCoverImage()]}
+     imageStyle={{opacity: isHeaderHide ? 0 : 1}}>
      <TouchableOpacity onPress={() => backScreen()} style={styles.backbutton}>
-       <MemoIcArrowBack width={normalize(24)} height={normalize(24)} />
+       {coverPath && !isHeaderHide ? (
+         <MemoIcArrowBackCircle width={normalize(32)} height={normalize(32)} />
+       ) : (
+         <MemoIcArrowBack width={normalize(24)} height={normalize(24)} />
+       )}
      </TouchableOpacity>
      <View style={styles.containerAction}>
        {!isFollow && isHeaderHide ? (
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->